chsdi/templates/htmlpopup/geocover_polygon.mako

```diff
@@ -6,10 +6,18 @@
     litho = 'litho_%s' % lang
     chrono = 'chrono_%s' % lang
     harmos_rev = 'harmos_rev_%s' % lang
+    if c['attributes']['litstrat_link'] is not None:
+            url = "http://" + c['attributes']['litstrat_link']
 %>
     <tr><td class="cell-left">${_('geocover_basisdatensatz')}</td><td>${c['attributes']['basisdatensatz'] or '-'}</td></tr>
     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geocover.description')}</td><td>${c['attributes']['description'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.swisstopo.geologie-geocover.litstrat_link')}</td><td>${c['attributes']['litstrat_link'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.swisstopo.geologie-geocover.litstrat_link')}</td><td>
+    % if c['attributes']['litstrat_link']:
+          <a href="${url}" target="_blank">${c['attributes']['litstrat_link'] or '-'}</a>
```

Comment:
Please make it
<a href="${url}" target="_blank">Link</a>
no need for the "or" clause since the attribute is only showed if it exists, am I right?